src/Security.tsx

```diff
@@ -87,11 +83,17 @@ const Security: React.FC<{
     return <OktaError error={err} />;
   }
 
-  if (oktaAuthMajorVersion !== process.env.AUTH_JS_MAJOR_VERSION 
-      // use SKIP_VERSION_CHECK flag to control version check in tests
-      && process.env.SKIP_VERSION_CHECK !== '1') {
-    const err = new AuthSdkError(`Passed in oktaAuth is not compatible with the SDK, okta-auth-js version ${process.env.AUTH_JS_MAJOR_VERSION}.x is the current supported version.`);
-    return <OktaError error={err} />;
+  if (oktaAuth && oktaAuth._oktaUserAgent) {
```

Comment:
This check will miss any version that does not have `_oktaUserAgent` (below 5.3.1). Maybe we can use a warn instead of throwing an error, so it won't break.